src/app/globals.css

```
@@ -582,6 +620,116 @@ body {
   color: var(--heading);
 }
 
+/* GitHub-flavored alerts (Markdown / MDX) + matching rST admonitions.
+ * One per-type accent variable drives both border, title color, and tinted
+ * background, so MDX `> [!NOTE]` and rST `.. note::` look visually identical.
+ * Colors track GitHub Primer's alert palette adapted to amytis CSS variables. */
+.alert {
+  --alert-accent: var(--accent);
+  margin: 2rem 0;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)